qiskit_nature/problems/sampling/protein_folding/penalty_parameters.py

```diff
@@ -0,0 +1,60 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Gathers parameters for penalty terms in a protein folding problem Hamiltonian."""
+
+
+class PenaltyParameters:
+    """Gathers parameters for penalty terms in a protein folding problem Hamiltonian."""
+
+    def __init__(
+        self,
+        penalty_chiral: float = 10,
+        penalty_back: float = 10,
+        penalty_1: float = 10,
+        penalty_contacts: float = 10,
```

Comment:
If these are floats, I would make that explicit by writing `10.0`. The type hint alone will not make Python interpret these as floats rather than ints.